tests/integration/backup/test_v3_backup.py

```diff
@@ -424,6 +424,105 @@ def test_roundtrip_from_file(
     assert data_file.read_bytes() == new_data_file.read_bytes()
 
 
+def test_roundtrip_from_file_skipping_topic_creation(
```

Comment:
question: Should we also test behavior when the topic doesn't exist? It seems likely we'll run into that.

Reply:
Nice, I'm happy with the test as it locks the behavior in 👍 It would be nice to have a more solid way to test this in the future, distinguishing between other reasons for timing out.